docs/development/doc_guide.rst

```diff
@@ -1,76 +1,829 @@
-************************
-Documentation Guidelines
-************************
+*********************
+Writing Documentation
+*********************
 
-This document describes the documentation requirements and guidelines
-to be followed during the development of PlasmaPy and affiliated
-packages.
+Documentation that is up-to-date and understandable is vital to the
+health of a software project. This page describes the documentation
+requirements and guidelines to be followed during the development of
+PlasmaPy and affiliated packages.
+
+.. hint::
+
+   Updating documentation is one of the best ways to make a first
+   contribution to an open source software project.
+
+.. note::
+
+   If you discover areas within PlasmaPy's documentation that are
+   confusing or incomplete, please `raise an issue`_! This really helps
+   PlasmaPy not only by helping us improve the documentation for all,
+   but also by creating opportunities for new contributors to make their
+   first contribution to the project.
+
+`PlasmaPy's documentation`_ is hosted by `Read the Docs`_ and is
+available at these locations:
+
+* The documentation corresponding to the most recent official release is
+  labelled ``stable`` and is found at `https://docs.plasmapy.org/`_ or
+  `https://docs.plasmapy.org/en/stable/`_.
+
+* The documentation corresponding to the ``main`` branch on `PlasmaPy's
+  GitHub repository`_ is labelled ``latest`` and can be found at
+  `https://docs.plasmapy.org/en/latest/`_.
+
+A preview of the documentation is generated every time a pull request is
+created or updated. You can access this preview by scrolling down to the
+checks at the bottom of a pull request, and clicking on ``Details`` next
+to ``docs/readthedocs.org:plasmapy``.
 
 Building documentation
 ======================
-Documentation is built from the main branch on every commit pushed
-to it.
 
-Sphinx, the documentation generator of PlasmaPy, uses reStructuredText (reST) as its markup language. A primer on reST is available at this `webpage
-<https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_
-of Sphinx's website.
+There are two methods for building the documentation: make_ and tox_.
+
+* Using make_ will build the documentation based off of what is in the
+  current directory structure. Using make_ is quicker for local builds
+  but requires that setting up all dependencies ahead of time.
+* Using tox_ does not require setting up all dependencies ahead of time,
+  but is more computationally intensive since it builds the package and
+  environment before building the documentation. Consequently, PlasmaPy
+  uses tox_ for building the documentation on continuous integration
+  testing platforms.
```

Comment:
```suggestion
* Using tox_ does not require setting up all dependencies ahead of time,
  but is more computationally intensive since it creates a virtual environment
  and builds the package before building the documentation. Consequently,
  PlasmaPy uses tox_ for building the documentation on continuous integration
  testing platforms.
```